From fd68f35dc9a94eb18113b0208cfcf276594787fe Mon Sep 17 00:00:00 2001
From: James Cloos <cloos@jhcloos.com>
Date: Thu, 14 Aug 2008 14:03:15 -0400
Subject: [PATCH 6/6] Do not install xcb-xlib.pc

With libxcb-xlib gone there is no need for xcb-xlib.pc.

Signed-off-by: James Cloos <cloos@jhcloos.com>
---
 Makefile.am |    3 +--
 1 files changed, 1 insertions(+), 2 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index 2bf2d0f..738c0c8 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -3,8 +3,7 @@ SUBDIRS=src tests doc
 pkgconfigdir = $(libdir)/pkgconfig
 
 pkgconfig_DATA = \
-xcb.pc \
-xcb-xlib.pc
+xcb.pc
 
 if BUILD_COMPOSITE
 pkgconfig_DATA += xcb-composite.pc
-- 
1.5.6.GIT

